src/sig_stfl/sig_stfl.c

```diff
@@ -920,20 +921,36 @@ OQS_API OQS_STATUS OQS_SIG_STFL_verify(const OQS_SIG_STFL *sig, const uint8_t *m
 	}
 }
 
+
 OQS_API OQS_STATUS OQS_SIG_STFL_sigs_remaining(const OQS_SIG_STFL *sig, unsigned long long *remain, const OQS_SIG_STFL_SECRET_KEY *secret_key) {
+#ifndef OQS_ALLOW_SFTL_KEY_AND_SIG_GEN
+	(void)sig;
+	(void)remain;
+	(void)secret_key;
+	return OQS_ERROR;
```

Comment:
Would it make sense to add an OQS_UNUSED macro in our common defines?

Likely doesn't make sense for the PR, and if this is the only spot in the codebase that has unused params it might not make sense at all...

Reply:
cc: @baentsch @dstebila @SWilson4 @praveksharma

Reply:
Sounds good to me. :)